codespell_lib/data/dictionary_usage.txt

```diff
@@ -2,7 +2,6 @@ black-hat->malicious actor, attacker,
 blackhat->malicious actor, attacker,
 blacklist->blocklist
 blacklists->blocklists
-blueish->bluish
```

Comment:
[OED](https://www.oed.com/search/dictionary/?q=blueish&tl=true) and other dictionaries show `blueish` as a valid variant of `bluish`.

Additionally, I can find `blueish` in both [en_US](http://app.aspell.net/lookup?dict=en_US;words=blueish) and [en_GB](http://app.aspell.net/lookup?dict=en_GB-ize;words=blueish) SCOWL lists.

Reply:
If this is an agreement to removing that incorrect "correction", can you please mark the conversation as resolved? An explanation for the removal is already in the commit message. Happy to add your explanations as well if you feel that's necessary.